### DISCUSSION TITLE
ID GNN Implementation

### DISCUSSION THREAD

**Discussion by joewilaj** [danielegrattarola/spektral#173] [General]
Hello,

I am searching for an implementation of ID aware GNNs  (Gomes-Selman,Leskovec, Ying, You, 2021, https://arxiv.org/pdf/2101.10320.pdf). It would be great if these were available with spektral.

**Comment by danielegrattarola** [comment]
Hi, 

thanks for the suggestion, I have put this on my TODO list and I will comment here once this is implemented. 

Cheers

**Comment by joewilaj** [comment]
That is awesome. Thank you
